account-kit/core/src/store/types.ts

```diff
@@ -95,11 +92,10 @@ export type StoreState = {
       >;
     }>;
   };
+  bundlerClient: ClientWithAlchemyMethods;
   // serializable state
   // NOTE: in some cases this can be serialized to cookie storage
   // be mindful of how big this gets. cookie limit 4KB
-  transport: AlchemyTransport;
```

Comment:
transport has been removed because it's not serializable in NextJS (makes sense it's a function). so instead we store connections as `transport.config` so that we can serialize it and create a transport when we need it. The `bundlerClient` is still fine to pass along it seems though

Reply:
Is there any reason to serialize the whole bundler client? Does it actually get recovered correctly from deserialization? I'm actually not aware of the state and stuff inside of it, but if there is it might make sense to be explicit about it?

Reply:
Ignore me saw the test above.

Reply:
Do you have concerns on us accidentally adding something to the bundler client that we need that will make this unserializable? I guess we have the E2E test and can block future updates and revisit then.

Reply:
the bundler client itself isn't being serialized anymore to storage, BUT i am concerned about the nextjs thing that I ran into. I need to figure out how best to test that the `config` returned from `createConfig` is serializable in the way that next js expects it to be. 

I'm gonna dig around a little bit and see if I can write a test that covers that (maybe nextjs exposes their serialization/deserialization methods and we can write a test explicitly for that so this doesn't become an issue in the future)

Reply:
update: the issue wasn't on the actual body of the config, but it's in the result of the deserialized cookie returned from `cookieToInitialState`

it means that I CAN have transport in the config, I just can't deserialize it to a function... I'll have to come back to that later